src/tests/unit/tests/reports/package/integration.test.ts

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 import { AxeResults } from 'axe-core';
 import { reporterFactory } from 'reports/package/reporter-factory';
+import { formatHtmlForSnapshot } from 'tests/end-to-end/common/element-snapshot-formatter';
```

Comment:
NIT: unit test should not take a dependency under end-to-end folder.

Reply:
Good point. I will follow up with moving this so it can be used more broadly.